src/angularComponent/ngcOmniboxController.js

```diff
@@ -200,6 +200,19 @@ export default class NgcOmniboxController {
     }
   }
 
+  /**
+   * Removes a particular suggestion item frmo the list of selected items.
```

Comment:
This is the reason why I like "deselect" better as a name -- because it's singular/plural agnostic. Also, the JSDoc you have written here expressly indicates that there is a "list of selected items", which is clearly not true if multiple select isn't on.